src/main/scala/tilelink/Broadcast.scala

```diff
@@ -16,9 +16,13 @@ class TLBroadcast(lineBytes: Int, numTrackers: Int = 4, bufferless: Boolean = fa
 
   val node = TLAdapterNode(
     clientFn  = { cp =>
-      cp.v1copy(clients = Seq(TLMasterParameters.v1(
-        name     = "TLBroadcast",
-        sourceId = IdRange(0, 1 << log2Ceil(cp.endSourceId*4)))))
+      cp.v1copy(
+        clients = Seq(TLMasterParameters.v1(
+          name     = "TLBroadcast",
+          sourceId = IdRange(0, 1 << log2Ceil(cp.endSourceId*4)))),
+        echoFields    = cp.echoFields,
+        requestFields = cp.requestFields,
+        responseKeys  = cp.responseKeys)
```

Comment:
I mean, it does not hurt to list these three lines, but ... they are copied by v1copy already anyway. So they are just dead code.

Reply:
You could probably have left the entire node unmodified.